resource/ckb.toml

```diff
@@ -199,3 +199,11 @@ block_uncles_cache_size    = 30
 # [indexer_v2]
 # # Indexing the pending txs in the ckb tx-pool
 # index_tx_pool = false
+# # Customize block filtering rules to index only retained blocks
+# block_filter = "block.header.number.to_uint() >= \"0x0\".to_uint()"
+# # Customize cell filtering rules to index only retained cells
+# cell_filter = "let script = output.type;script!=() && script.code_hash == \"0x00000000000000000000000000000000000000000000000000545950455f4944\""
+# # The initial tip number and hash can be set as the starting height for building indexes in indexer-r. Effective only during the initial index creation.
+# init_tip_number = 10000
+# # The initial tip hash must match the tip number; otherwise, it will result in a rollback to empty.
+# init_tip_hash = "0x8fbd0ec887159d2814cee475911600e3589849670f5ee1ed9798b38fdeef4e44"
```

Comment:
suggest to use one block hash parameter only, indexer may use `get_header` rpc to fetch the block number, and start the indexing service only when corresponding header was found.

Reply:
Entering a hash manually has the added feature that the block height of the node itself may not have been reached yet and `get_header` may fail.

Reply:
Entering a number manually considering a situation where the block height of the node itself may not have been reached and get_header may fail.

Reply:
please refer to my previous comment:
> start the indexing service only when corresponding header was found.

Reply:
Another issue is that if the hash is wrong, then there is no chance of rollback.

Reply:
indexer service won't starting if the hash is wrong.

Reply:
OK, got it.